### DISCUSSION TITLE
Writing comments under the root `template` tag causes route jump exception

### DISCUSSION THREAD

**Discussion by A81N9** [vuejs/router#2104] [Help and Questions] [answered]
### Reproduction

Writing comments under the root `template` tag causes route jump exception

### Steps to reproduce the bug

I found that the route could not jump normally. Finally, I found out that it was caused by writing comments under the root `template` tag as follows:

![image](https://github.com/vuejs/router/assets/53040248/376c89c9-a427-4fc3-b68e-6b4301cb7c56)

Routes jump normally if I write as follows

![image](https://github.com/vuejs/router/assets/53040248/57ad3778-6021-4361-9102-0611bf9cb0ee)

### Expected behavior

/

### Actual behavior

/

**Comment by posva** [comment, marked as answer]
If you are using a transition, this is inherent to transition

**Reply by A81N9** [reply]
Thank you for your reply
